sql/gen-sql-markdown.py

```diff
@@ -15,12 +15,18 @@
 # limitations under the License.
 #
 
-import sys
 import os
+import re
+import sys
 from collections import namedtuple
+from textwrap import dedent
+
+from markdown import markdown
```

Comment:
@nchammas, I know it's a bit hacky but what do you think about we don't add more dependencies here by using `markdown` under `mkdocs` directly? e.g. `from mkdocs.structure.pages.markdown import markdown`. I checked this package from version 1.0.0 ~ master branch. Seems pretty stable.

I was thinking it's better to explicitly avoid adding more dependencies. I know `mkdocs` is dependent on `markdown` but different versions are required (see https://github.com/mkdocs/mkdocs/blob/399f8428b84c4a91a00e0388d7e9bb5260dc8ec3/setup.py#L61 as an example).

In this way, we can also avoid updating `dev/create-release/spark-rm/Dockerfile` as well.

Reply:
Ugh, that's pretty gnarly, but sure.